content/tracking/setup.md

```diff
@@ -0,0 +1,133 @@
+
+---
+
+#### **Step 2:** Add the Snowplow JavaScript snippet
+
+Add the below snippet to all of the pages you would like to use Snowplow tracking. **Make sure to update the {{Link to the sp.js file}} variable.**
+
+Place the `<script>` tag into the `<head>` element of your pages:
+
+```html
+<script type="text/javascript">
+  (function (p, l, o, w, i, n, g) {
+    if (!p[i]) {
+      p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
+      p.GlobalSnowplowNamespace.push(i);
+      p[i] = function () {
+        (p[i].q = p[i].q || []).push(arguments);
+      };
+      p[i].q = p[i].q || [];
+      n = l.createElement(o);
+      g = l.getElementsByTagName(o)[0];
+      n.async = 1;
+      n.src = w;
+      g.parentNode.insertBefore(n, g);
+    }
+  })(window, document, "script", "{{Link to sp.js file}}", "snowplow");
+</script>
+```
+
+---
+
+#### **Step 3:** Configure the global `tracker` instance
+
+You can now create the `newTracker`, with the following arguments. This creates an instance of a basic tracker without any additional context.
+
+- Tracker Name: `'sp'`
+- Collector Url: `'{{Url for Collector}}'`
+
+```javascript
+window.snowplow("newTracker", "sp", "{{Url for Collector}}", {
+  /* tracker options */
+});
+```
+
+In addition to the basic tracker, you can add any number of options for using Snowplow's more advanced features. <a target="_blank" href="https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/">See more on our documentation</a>.
+
+#### **Step 4:** Configure the tracker to use the `WebVitalsPlugin`
+
+To add the `WebVitalsPlugin` on the JavaScript tracker and enable web performance data collection, you should include it as shown below:
+
+```javascript
+window.snowplow(
+  "addPlugin:sp",
+  "https://cdn.jsdelivr.net/npm/@snowplow/browser-plugin-web-vitals@3/dist/index.umd.min.js",
+  ["snowplowWebVitals", "WebVitalsPlugin"]
+);
+```
+
+The `WebVitalsPlugin` can accept some additional options which are described in detail on the plugin [documentation page](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/web-vitals/).
+
+Now the tracker has everything required to start collecting web performance data.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### The `web_vitals` event
+
+After adding the web-vitals plugin to your setup, the tracker will send a `web_vitals` event every time a user navigates away from one of your pages. This event will include as accurate information about web vital metrics as possible. To see which data are collected in this event, you can browse the relevant [iglu schema](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/web_vitals/jsonschema/).
```

Comment:
There's no harm in mentioning what metrics we collect here (to save the user having to parse this info out of the JSON), this will be:

- Cumulative layout shift
- First input delay
- Largest contentful paint
- First contentful paint
- Interaction to next paint
- Time to first byte

Reply:
> Is this true?
It is true 😂 

> There's no harm in mentioning what metrics we collect here
👍  Adding